app/views/users/_form.html.erb

```diff
@@ -46,7 +45,7 @@
   
   
       <div class="form-group">
-        <% if @user.crypted_password.nil? %>
+        <% if :user.crypted_password.nil? %>
```

Comment:
use @user.crypted..... don't change this line

Reply:
never change @ to : again. It will not help us. It will break the stuff even more

Reply:
Ok got it